cgp/individual.py

```diff
@@ -106,9 +107,14 @@ def __init__(self, fitness: Union[float, None], genome: Genome) -> None:
     def __repr__(self):
         return f"Individual(idx={self.idx}, fitness={self.fitness}, genome={self.genome}))"
 
-    def clone(self) -> "IndividualSingleGenome":
+    def clone(
+        self,
+        *,
```

Comment:
What does this `*` argument do here?

Reply:
this is just the notation for keyword-only arguments (https://www.python.org/dev/peps/pep-3102/). everything behind `*` needs to be passed like `argument_name=value`